src/logic/calculate.test.js

```diff
@@ -0,0 +1,16 @@
+import calculate from './calculate';
+
+it('should return object and button name', () => {
+  const object = {
+    total: '',
+    operation: '',
+    next: '0',
+  };
+
+  const obj = {
+    total: null,
+    next: '0',
+    operation: null,
+  };
+  expect(calculate(object, 'AC')).toStrictEqual(obj);
+});
```

Comment:
- [ ] Kindly add to your `calculate.test.js` some tests for addition, multiple, modules, division, subtraction, and some signs like `.`, `=`.

For example:
```  
test('calculate  6÷2=3', () => {
    expect(calculate({ total: '6', next: '2', operation: '÷' }, '=')).toStrictEqual({ total: '3', next: null, operation: null });
  });
```